src/test/java/seedu/address/storage/XmlAdaptedExpenseTest.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.storage;
+
+import static org.junit.Assert.assertEquals;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeParseException;
+import java.util.HashMap;
+
+import org.junit.Test;
+
+import seedu.address.model.Menu;
+import seedu.address.model.expenses.Expense;
+import seedu.address.model.expenses.ExpenseType;
+import seedu.address.testutil.Assert;
+
+public class XmlAdaptedExpenseTest {
+    private static final String INVALID_ITEM = "1";
+    private static final String INVALID_DATETIME = "1/1/2018 12:34:56";
+
+    private static final String VALID_ITEM = "RS01";
+    private static final String VALID_DATETIME = "2018-10-14T22:34:40.283201400";
+
+    private static final Menu VALID_MENU = new Menu();
+    {
```

Comment:
JUnit provides `@Before` tag for initializing elements used for the test. Just putting it here, in case it is required somewhere down the road. Initializers like this one here seems to work, but it seems to be frowned upon in the context of this module. Consider changing to `@Before`.